pkg/rehearse/jobs_test.go

```diff
@@ -648,6 +665,16 @@ func TestExecuteJobsPositive(t *testing.T) {
 	anotherTargetOrg := "anotherOrg"
 	anotherTargetRepo := "anotherRepo"
 	testCiopConfigs := generateTestConfigFiles()
+	compressedJob1, err := gzip.CompressString(testingCiOpCfgJob1YAML)
+	if err != nil {
+		t.Fatalf("Failed to compress config: %v", err)
+	}
+	job1Cfg := base64.StdEncoding.EncodeToString([]byte(compressedJob1))
+	compressedJob2, err := gzip.CompressString(testingCiOpCfgJob2YAML)
+	if err != nil {
+		t.Fatalf("Failed to compress config: %v", err)
+	}
+	job2Cfg := base64.StdEncoding.EncodeToString([]byte(compressedJob2))
```

Comment:
Can we keep one uncompressed to make sure that continues to work?

Reply:
That variable is the expected config_spec output for that function, so it will always be compressed.